schema/release-schema.json

```diff
@@ -114,7 +114,7 @@
       "items": {
         "$ref": "#/definitions/RelatedProcess"
       },
-      "description": "The details of related processes: for example, if this process follows on from one or more other processes, represented under a separate ocid. This is commonly used to relate mini-competitions to their parent frameworks or individual tenders to a broader planning process.",
+      "description": "The details of related processes: for example, if this process follows on from one or more other processes, represented under a separate ocid. This field is commonly used to connect contracting and planning processes and to connect different stages of a multi-stage procedure, like a framework agreement procedure.",
```

Comment:
```suggestion
      "description": "References to related, preceding contracting (or planning) processes. This information concerns the contracting process. For example, the contracting process may refer to its planning process(es). In multi-stage procedures (e.g. framework agreements with reopening of competition), the contracting process for a later stage may refer to the contracting process for the first stage.",
```

Reply:
Could you clarify what you mean by "This information concerns the contracting process"? I'm not sure if you mean "this field should only be used in the context of a contracting process" or "the codes describe the relationship between this contracting process and the related processes".

Reply:
That phrase is used throughout the schema to describe which information is for the planning process, contracting process or either.